upup/pkg/fi/cloudup/urls.go

```diff
@@ -66,7 +66,7 @@ func BaseUrl() (*url.URL, error) {
 	} else {
 		kopsBaseUrl, err = url.Parse(baseUrlString)
 		if err != nil {
-			return nil, fmt.Errorf("unable to parse env var KOPS_BASE_URL %q as an url: %v", baseUrlString, err)
+			return nil, fmt.Errorf("unable to parse env var KOPS_BASE_URL %q as a url: %v", baseUrlString, err)
```

Comment:
If we want to be picky ... url is URL

Reply:
Thanks for your reply. https://itknowledgeexchange.techtarget.com/writing-for-business/which-is-correct-a-url-or-an-url/